packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/order/choice/choice.html

```diff
@@ -12,7 +12,7 @@
 
     <div data-ng-if="!$ctrl.isStepLoading">
         <div
-            data-ng-if="($ctrl.phoneOffers.length > 1 || $ctrl.merchandise.length)"
+            data-ng-if="($ctrl.phoneOffers.length > 0 || $ctrl.merchandise.length)"
```

Comment:
To have same tests : 
```suggestion
            data-ng-if="($ctrl.phoneOffers.length || $ctrl.merchandise.length)"
```